src/main/java/com/labsynch/labseer/chemclasses/bbchem/ChemStructureServiceBBChemImpl.java

```diff
@@ -592,11 +590,14 @@ public boolean standardizedMolCompare(String queryMol, String targetMol) throws
 
 	@Override
 	public boolean isEmpty(String molFile) throws CmpdRegMolFormatException {
-		RWMol mol = bbChemStructureService.getPartiallySanitizedRWMol(molFile);
-		Boolean hasAtoms =  mol.getNumAtoms() == 0.0;
-		Boolean hasBonds =  mol.getNumBonds() == 0.0;
-		Boolean hasSGroups = RDKFuncs.getSubstanceGroupCount(mol) == 0.0;
-		return !hasAtoms && !hasBonds && !hasSGroups;
+		try{
+			HttpURLConnection mol = bbChemStructureService.postToPreprocessorService(molFile);
+			return mol.getResponseCode() == 4004;
```

Comment:
Should this be 404, not 4004? Never heard of a 4004 response code

Reply:
Yeah, uh this was totally wrong.

A) the function is calling process service but its called `postToPreprocessorService`
B) The response shouldn't have been a status code check, the actual json has the `4004` code in it.

I need to refactor a few things to get this right.

Reply:
Refactored this.